server/models/handlers/account.js

```diff
@@ -1,22 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
-        type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
-        studentId: '20764242',
-        firstname: 'Victor',
-        lastname: 'Hugo',
-        email: 'victorhugo@gmail.com',
-        phone: '6476442200'
+async function createUserAccount(firstName, lastName, type, studentId, email, password, phone, userType, workerId, specialization, schoolId) {
+    // Searches the student table to see if a student account exists for the given data.
+    var user;
+    const accessToken = new UIDGenerator(UIDGenerator.BASE16, 20);
```

Comment:
Hi Nancy,
I ran the automated test code and there was test failure as UIDGenerator was not imported (shown below). I would recommend importing that method by first installing that dependency in the `server` folder and then adding the appropriate import statement. Refer to this link for detailed steps: https://www.npmjs.com/package/uid-generator
![image](https://user-images.githubusercontent.com/32720322/98119140-32ad6580-1e7a-11eb-84a2-4e5c7787372e.png)

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I have installed the UID generator using npm which generates access tokens  with custom size and base-encoding.
<img width="733" alt="Screen Shot 2020-11-04 at 8 53 27 AM" src="https://user-images.githubusercontent.com/71143782/98119875-37264e00-1e7b-11eb-922d-ebf2ef33fb7e.png">
Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
I am currently facing a lot of issues when running your code. I would recommend jumping on a call to discuss and work on the required changes together.